src/levels/mixed/describe.js

```diff
@@ -1,4 +1,4 @@
-﻿exports.level = {
```

Comment:
whats this change for?

Reply:
It marks this line as deleted and added. I could not find out what has been changed in this line. Maybe it is something with whitespace ...